README.md

```diff
@@ -44,18 +42,18 @@ $ npm install textlint -g
 - Node.js 4.0.0+
 - npm 2.0.0+
 
-Test: Run `node -v` in your console. The version should be higher than v4.0.0.
+If you're not sure what version of Node you're running, you can run `node -v` in your console to find out. TextLint works on versions v4.0.0 and up.
 
-:warning: Caution: Mixed location of installation.
+:warning: Note:
 
-- If you have installed `textlint` as `--global`(`-g`), must install each rule as `--global`.
-- If you have installed `textlint` as `--save-dev`(`-D`), must install each rule as `--save-dev`.
+- If you have installed `textlint` globally (e.g. `npm install textlint --global`), you must install each rule globally as well `--global`.
```

Comment:
https://ci.appveyor.com/project/azu/textlint/build/1.0.865#L491
[rousseau](https://github.com/GitbookIO/rousseau) warn "readbility".

The sentence may be too long.